packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -1251,7 +1225,10 @@ export class IndexedDbLruDelegate implements ReferenceDelegate, LruDelegate {
               // Our size accounting requires us to read all documents before
               // removing them.
               return changeBuffer.getEntry(txn, docKey).next(() => {
-                changeBuffer.removeEntry(docKey);
+                changeBuffer.removeEntry(
+                  docKey,
+                  SnapshotVersion.forDeletedDoc()
+                );
```

Comment:
Per chat conversation, I will not stand for removeEntry() requiring a version that is MEANINGLESS!!! j/k...  But I am realizing that there is exactly one place where we want to track removed entries and that is for failed limbo resolution, so I'm wondering if we should:

1. Remove the `trackRemovals` option.
2. Make it so removeEntry() doesn't _require_ a version.
3. Let you optionally pass an entry in.  This could be `removeEntry(key, version)` or if we want to make it a little more explicit (and borrow a bit from the existing shape of the API), maybe `removeEntry(key, {trackRemovalVersion: version})` or something.

Alternatively, since this such a specific case, we could consider _not_ storing the sentinel delete and inventing a localStorage notification for it instead.